nsxt/resource_nsxt_policy_group_test.go

```diff
@@ -60,6 +61,50 @@ func TestAccResourceNsxtPolicyGroup_singleIPAddressCriteria(t *testing.T) {
 	})
 }
 
+func TestAccResourceNsxtGlobalPolicyGroup_singleIPAddressCriteria(t *testing.T) {
```

Comment:
Can we reuse all LM tests to run for GM also? It is important to test all scenarios since group is a complex resource.

Reply:
Yes, current LM tests are already run for GM. This one is just to test Group resource works for specific domain.